packages/block-library/src/post-terms/block.json

```diff
@@ -23,7 +23,11 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
Post terms are rendered as a list of comma separated links, so the text color only updates the comma. Should we add `link` as one of the defaults?